core/test/functional/routes/api/users_spec.js

```diff
@@ -38,7 +38,9 @@ describe('User API', function () {
     describe('As Owner', function () {
         describe('Browse', function () {
             it('returns dates in ISO 8601 format', function (done) {
-                request.get(testUtils.API.getApiQuery('users/'))
+                //@TODO: postgres returns for default oder (last_login DESC) something else then sqlite
+                //@TODO: maybe related to https://github.com/TryGhost/Ghost/issues/6104
+                request.get(testUtils.API.getApiQuery('users/?order=id%20ASC'))
```

Comment:
@ErisDS FYI
The test has changed, because we now have more users in the database (see `users:roles:no-owner`) - and then the test wasn't working anymore, because of different return of order for different databases.
I fixed it with order=id%20ASC for now and i wrote a `TODO` which points maybe to the correct issue.